doc/nrf/gs_installing.rst

```diff
@@ -535,20 +524,15 @@ If you chose to use |SES| for :ref:`building and programming a sample applicatio
 
    a. Navigate to the :file:`bin` directory.
    #. Run the :file:`emStudio` executable file.
-   #. Select :guilabel:`File` > :guilabel:`Open nRF Connect SDK Project`.
-
-      .. figure:: images/ses_open.png
-         :alt: Open nRF Connect SDK Project menu
-
-         Open nRF Connect SDK Project menu
+   #. Set the GNU ARM Embedded Toolchain directory to :file:`c:/gnuarmemb` after clicking :guilabel:`OK` on the pop-up window that opens.
+   #. At this stage, you can close the :guilabel:`nRF Connect options` window.
```

Comment:
No sure if "at this stage, you can" is needed here. "At this stage" is synonymous with a new step IMO.